README.md

```diff
@@ -1,5 +1,5 @@
 <p align="center">
-<img src="https://i.imgur.com/bAZFAGF.png" width="450" height="243" alt="Kadena" title="Kadena">
+<img src="https://raw.githubusercontent.com/kadena-community/kadena.js/main/common/images/Kadena.JS_logo-black.png" width="450" height="243" alt="Kadena" title="Kadena">
```

Comment:
2 things: 

1. Let's host the logo in this repository instead of linking to a community project
2. Let's use the good green logo :)

Reply:
![Full Kadena Wordmark](https://github.com/kadena-io/pact/assets/53714543/d6edabaa-03d5-4e11-8f51-e37987e91336)

Are you able to see this?

Reply:
Yeah i can see that :)